src/topology/algebra/module.lean

```diff
@@ -515,6 +522,12 @@ variable (M)
   .. linear_equiv.refl R M }
 end
 
+@[simp, elim_cast] lemma coe_refl :
+  ((continuous_linear_equiv.refl M : M ≃L[R] M) : M →L[R] M) = continuous_linear_map.id := rfl
```

Comment:
Recently Kevin made the ring explicit in the definition of `linear_equiv.refl`, and this makes sense because it can not be inferred from anything. Could you also make it explicit here?